### PR TITLE
More config improvements

### PR DESCRIPTION
**Pull request by benbc**

### CODE REVIEW

**Comment by benbc** [comment]
@pe4cey Here is my part of the webadmin cleanup. Please can you review it?

**Comment by benbc** [comment]
@jjaderberg or @mariascharin: please take a look at the docs part of this.

File: manual/contents/src/operations/configuration.asciidoc [hunk 1]
```diff
@@ -13,10 +13,6 @@ include::{importdir}/neo4j-kernel-docs-jar/ops/introduction.asciidoc[]
 
 :leveloffset: 2
 
-include::{importdir}/neo4j-server-docs-jar/ops/server-configuration.asciidoc[]
```
Comment (jjaderberg) [inline manual/contents/src/operations/configuration.asciidoc:16]: Have you also removed the _server-configuration.asciidoc_ source file?

**Comment by jjaderberg** [comment]
Looks good! This is labeled `NOT READY FOR MERGE`, are you waiting for Mark's PR to merge them together?

File: community/kernel/src/docs/ops/configuration.asciidoc [hunk 2]
```diff
@@ -16,6 +14,8 @@ For additional configuration settings, see:
 * <<config-org.neo4j.ext.udc.UdcSettings>>
 * <<config-org.neo4j.metrics.MetricsSettings>>
 
-
 include::configuration-attributes.asciidoc[]
 
+include::{importdir}/neo4j-server-docs-jar/ops/configuration-attributes.asciidoc[]
+
+include::{importdir}/neo4j-dbms-docs-jar/ops/configuration-attributes.asciidoc[]
```
Comment (jjaderberg) [inline community/kernel/src/docs/ops/configuration.asciidoc:21]: This is probably right, but need to verify the header 'leveloffset' in the output, that the newly included sections are nested at the correct level.

File: manual/contents/src/operations/configuration.asciidoc (hunk 1 above)
Comment (benbc) [inline manual/contents/src/operations/configuration.asciidoc:16]: No! I forgot. Thank you.

**Comment by benbc** [comment]
@jjaderberg: I left it as "not ready for merge" because it needs a manual check of the rendered manual.

File: community/kernel/src/docs/ops/configuration.asciidoc (hunk 2 above)
Comment (benbc) [inline community/kernel/src/docs/ops/configuration.asciidoc:21]: Will do.

**Comment by pe4cey** [comment]
@benbc lgtm

**Pull request merged**